History.md

```diff
@@ -1,5 +1,18 @@
 # Liquid Change Log
 
+## 5.0.1 / 2021-03-24
+
+### Fixes
+* Add ParseTreeVisitor to Echo tag (#1414) [CP Clermont]
+* Fix Rubocop (#1402) [Dylan Thacker-Smith]
+* Add a hint to sign CLA only if it's the first time (#1406, #1405) [ADTC]
```

Comment:
I think the change log is more so for the users of the library, rather than for contributors, so I think we could remove these.

```suggestion
```